source/less/blocks/examples.less

```
@@ -0,0 +1,48 @@
+.main-container__examples.examples {
```

Не используй в CSS двойные классы без необходимости.
```suggestion
.examples {
```